lib/config/executor/container_vm.go

```diff
@@ -108,7 +123,7 @@ type ContainerVM struct {
 // in that there is no process inherently associated - this is closer to a ThreadPool than a Thread and
 // is the owner of the shared filesystem environment. This is the guest visible complement to ContainerVM.
 type ExecutorConfig struct {
-	Common `vic:"0.1" scope:"read-only" key:"common"`
```

Comment:
So.. what happened to `Common` here? Or was `Common` renamed to `CommonSpecForVM` everywhere? Because if it is renamed everywhere, why was it renamed? I am skeptical that this longer name is any more semantically useful given the context of the object than the original, shorter name.